plugins/Morpheus/stylesheets/base/mixins.less

```diff
@@ -62,6 +62,14 @@ strong {
   }
 }
 
+.vertical-scroll-shadows(@bg-color: @color-white) {
+  background: linear-gradient(@bg-color 30%, fade(@bg-color, 0)) center top no-repeat local,
+    linear-gradient(fade(@bg-color, 0), @bg-color 70%) center bottom no-repeat local,
+    radial-gradient(farthest-side at 50% 0, fade(@color-silver, 60%), fade(@color-silver, 0%) ) center top no-repeat scroll,
+    radial-gradient(farthest-side at 50% 100%, fade(@color-silver, 60%), fade(@color-silver, 0%) ) center bottom no-repeat scroll;
+  background-size: 100% 40px, 100% 40px, 100% 14px, 100% 14px;
+}
```

Comment:
ℹ️ This code is not really easy to read and pretty reusable, so I moved it to a mixin.